packages/botfuel-dialog/tests/adapters/messenger-adapter.test.js

```diff
@@ -87,3 +95,126 @@ describe('MessengerAdapter', () => {
     expect(extended).toHaveProperty('payload.value', 'message');
   });
 });
+
+describe('MessengerAdapter request handling', () => {
+  test('should handle a request from messenger', async () => {
+    // Data definitions
+
+    const profile = {
+      firstName: 'Sample',
+      lastName: 'User',
+    };
+    const userMessageUuid = '254a66a3-9282-4e4f-bd86-e18a376d5bc4';
+    const botMessageUuid = 'cbad2bc9-070c-4b37-b4be-190904fc506e';
+    const handlingTimestamp = 1526398451040;
+    const responseTimestamp = 1526477455343;
+    const userMessageText = 'hello ca va?';
+    const testRequest = {
+      body: {
+        object: 'page',
+        entry: [
+          {
+            id: '<PAGE_ID>',
+            time: 1458692752478,
+            messaging: [
+              {
+                sender: {
+                  id: TEST_USER,
+                },
+                recipient: {
+                  id: '1803703159928348',
```

Comment:
can we put anything here?

Reply:
I replaced it by something more explicit.